src/Blocks/contact-form/block.json

```diff
@@ -7,6 +7,7 @@
 	"category": "widgets",
 	"description": "Gutenberg block for Fame lahjoitukset backend.",
 	"parent": ["famehelsinki/donation-form"],
+	"allowedBlocks": ["core/group"],
```

Comment:
I think we should have discussion if this is the best way to achieve this, since core/group allows all blocks as children.